response_operations_ui/views/collection_exercise.py

```diff
@@ -92,26 +96,30 @@ def build_collection_exercise_details(short_name: str, period: str, include_ci=T
     }
 
     if include_ci:
-        collection_instruments = collection_instrument_controllers.get_collection_instruments_by_classifier(
-            collection_exercise_id=collection_exercise_id, survey_id=survey_id
+        exercise_dict["collection_instruments"] = _build_collection_instruments_details(
+            collection_exercise_id, survey_id
         )
+    return exercise_dict
 
-        eq_ci_selectors = collection_instrument_controllers.get_collection_instruments_by_classifier(
```

Comment:
Used in one place, but call on everything including SEFT which is clearly wrong. Moved higher